package.json

```diff
@@ -5,52 +5,29 @@
   "author": "Antony Jones",
   "license": "MIT",
   "scripts": {
-    "dev": "vite dev",
-    "build": "vite build",
+    "dev": "vite dev --host",
+    "build": "vite build --config vite.components.config.js",
     "preview": "vite preview",
-    "package": "svelte-kit sync && svelte-package && publint",
-    "prepublishOnly": "npm run package",
     "check": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json",
     "check:watch": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json --watch",
     "lint": "eslint .",
     "test": "mocha \"src/**/*.spec.js\""
   },
-  "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "svelte": "./dist/index.js"
-    },
-    "./Banner.svelte": "./dist/Banner.svelte",
-    "./style.css": "./dist/style.css"
-  },
   "files": [
-    "dist",
-    "!dist/**/*.spec.*"
+    "src/lib/sdk",
+    "dist/v1/sdk"
   ],
-  "peerDependencies": {
-    "svelte": "^3.54.0 || ^4.0.0"
-  },
   "dependencies": {
     "js-cookie": "^3.0.1"
   },
   "devDependencies": {
-    "@beyonk/eslint-config": "^7.1.1",
-    "@hapi/code": "^8.0.2",
-    "@sveltejs/adapter-auto": "^2.0.0",
-    "@sveltejs/kit": "^1.24.1",
-    "@sveltejs/package": "^2.0.0",
-    "eslint": "^8.28.0",
-    "eslint-plugin-svelte3": "^4.0.0",
-    "mocha": "^8.3.0",
-    "publint": "^0.1.9",
-    "svelte": "^3.55.1",
-    "svelte-check": "^3.0.1",
-    "tslib": "^2.4.1",
-    "typescript": "^5.0.0",
-    "vite": "^4.2.0"
+    "@sveltejs/adapter-auto": "^2.1.0",
+    "@sveltejs/kit": "^1.22.4",
+    "@sveltejs/vite-plugin-svelte": "^2.4.4",
+    "svelte": "^4.0.5",
+    "vite": "^4.4.2"
   },
-  "svelte": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "svelte": "./dist/v1/sdk/index.js",
```

Comment:
I think this goes away in favour of an exports map:

exports: {
  "./": {
    "svelte": "src/lib/Banner.svelte",
    "import": "src/lib/Banner.svelte"
  },
  "./banner.css": {
    "import": "src/lib/banner.css"
  }
}

or something along those lines, I forget the exact syntax.

Reply:
Resolved 6e7380ecb1aff92a6fb16a2f9ae2f515d6480a09